### ISSUE TITLE
Erroneous link in readme.md

### ISSUE THREAD

**Issue by ffalcon31415** [spyoungtech/FreeSimpleGUI#4]
Last paragraph of the readme has link to submit an issue: 

If you still have a question or need help... no problem... help is available to you, at no cost. Simply [file an Issue](http://issues.freesimplegui.org/) on the FreeSimpleGUI GitHub repo and you'll get help.

Corrected link I think would be 
[file an Issue](https://github.com/spyoungtech/FreeSimpleGUI/issues)

*Edit* Hmm now I see there are other links that are not applicable any longer. I don't have time now but will try to look at those later.
btw thank you for creating this fork it is seriously appreciated.

**Comment by spyoungtech** [comment]
I've registered the corresponding domains and the links will work, eventually :) 

There's currently still a lot of things _not quite_ in place yet, especially with respect to documentation. I hope to straighten all that out in the near future.

**Comment by dimmv2** [comment]
hello , i have an question about a real bug in sdk , will it be fixed ?(example : bad variable for button or something like that)  or here the last package has only last stable version of pysimplegui ?

**Comment by spyoungtech** [comment]
@dimmv2 yes, bug reports will be accepted and, as bug fixes become available, they will be published in new versions. This is a fully supported fork, not just a snapshot. If you are having an issue with a bug, please open a separate issue.

**Comment by dimmv2** [comment]
> @dimmv2 yes, bug reports will be accepted. As bug fixes and other enhancements become available, they will be published in new versions. This is a fully supported fork, not just a snapshot. If you are having an issue with a bug, please open a separate issue.

Great news,thx for reply.

**Comment by caco3** [comment]
Also, the Support section ( https://github.com/spyoungtech/FreeSimpleGUI?tab=readme-ov-file#support- ) points to http://www.freesimplegui.org/ wich does not (yet) exist.

**Comment by DeflateAwning** [comment]
Also, this link in the README is broken:
```
calls.freesimplegui.org
```

**Comment by ffalcon31415** [comment]
> Also, the Support section ( https://github.com/spyoungtech/FreeSimpleGUI?tab=readme-ov-file#support- ) points to http://www.freesimplegui.org/ wich does not (yet) exist.

See @spyoungtech 's response to the initial issue posted above:

Thanks for the report. I've registered the corresponding domains and the links will work, eventually :)
```
There's currently still a lot of things not quite in place yet, especially with respect to documentation. I hope to straighten all that out in the near future.
```

**Comment by DeflateAwning** [comment]
Very exciting! Thanks for the updates!

**Comment by ffalcon31415** [comment]
Issue already closed by @spyoungtech ... nothing new.

**Comment by ffalcon31415** [comment]
Sorry I misread the history of this issue

**Comment by spyoungtech** [comment]
As an aside, full documentation is still available on readthedocs: https://freesimplegui.readthedocs.io/en/latest/
